blockonomics-woocommerce.php

```diff
@@ -582,7 +582,7 @@ function bnomics_async_scripts($url)
 add_action('admin_notices', 'blockonomics_plugin_activation');
 
 global $blockonomics_db_version;
-$blockonomics_db_version = '1.3';
+$blockonomics_db_version = '1.4';
```

Comment:
Incrementing db version ensures the fix executes for users who have already updated, and may be experiencing issues. 
We can consider splitting blockonomics_db_version and blockonomics_page_version. This will add multiple calls to db, but will make the versioning more clear.